client/python/README.md

```diff
@@ -1,19 +1,73 @@
-# Armada Python client
+Armada Python client
+=
+
+# Installation:
+
+On Linux Machine
+=
+1) Open your favorite shell, for example, good old [Bourne Again SHell, aka, bash](https://www.gnu.org/software/bash/), the somewhat newer [Z shell, aka, zsh](https://www.zsh.org/), or shiny new [fish](https://fishshell.com/).
+2) Install [Git](https://git-scm.com/) by running `sudo apt-get install git-all` on [Debian-based](https://www.debian.org/) distributions like [Ubuntu](https://ubuntu.com/), or sudo dnf install git on [Fedora](https://getfedora.org/) 
+   and closely-related [RPM-Package-Manager](https://rpm.org/) based distributions like [CentOS](https://www.centos.org/). For further information see [Installing Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
+3) Fork the G-Research Armada repository to your personal repository.
+
+4) Clone the forked armada repository on your personal git repository by running git clone [Armada Repo](https://github.com/G-Research/armada.git) and navigate into the new directory Armada project by running cd armada.
+
+### Python / Pyenv and Poetry Installation
+#### Note that this repository does not support python 3.10 for now
```

Comment:
This information is going to bit-rot here. Lets keep information about supported python versions in the metadata files that are more likely to be kept up to date over time.

Reply:
I stopped reviewing this PR for specifics at this point -- I think the direction is a little strange and we need to be explicit about what the goals are before I can review it further.

Reply:
@jayofdoom - how about suggesting a direction rather than throwing up your hands here? We're supposed to be supporting the fellows, not just tearing down their commits. We're not here to treat them like 20 year veterans that ought to be able to sort out a path with minimal inputs. If you don't have something constructive to say here, perhaps you should refrain from commenting on their pull requests.

Reply:
I have reached out to Precious this morning in Slack to figure this out. I didn't want to assume there was no intended direction and instead wanted to see what her goal was before making concrete suggestions. I almost always follow up PR feedback of this nature with a conversation when it's clear there's no meeting of the minds on what the goal of a change is.